index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        
+        <nav class="nav">
+          <a href="index.html">My logo</a>
+        <div class="menu">
+          <img src="/images/Union1.png" alt="">
+        </div>
+        </nav>
+    </header>
+    <section>
+    <div class="header-desc">
+        <h1>Hey There.
+        I’m Antoh.</h1>
+        <h3>I'm a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+        <div class="social">
+          <ul>
+            <li><a href="https://github.com/boniface0"><img src="/images/github (1).svg" alt=""></a></li>
+            <li><a href="https://www.linkedin.com/in/boniface-maingi-20a684251/"><img src="/images/Linkedin icon.svg" alt=""></a></li>
+            <li><a href=""><img src="/images/angel list.svg" alt=""></a></li>
+            <li><a href="https://twitter.com/Bony29906115/"><img src="/images/twitter.svg" alt=""></a></li>
+            <li><a href=""><img src="/images/medium.svg" alt=""></a></li>
+          </ul>
+        </div>
+        <div class="div">
+          <img src="/images/illustration-header-mobile.png" alt="">
+        </div>
```

Comment:
- [OPTIONAL] I highly remommend to always render the background images on the page using the CSS `background-image` property in order to have a clean and professional codebase.